### ISSUE TITLE
Needs "use lib '.';" in Build.PL for modern Perl versions

### ISSUE THREAD

**Issue by benkasminbullock** [syohex/p5-Text-AAlib#2]
With Perl 5.32 this fails to find builder/MyBuilder.pm, it needs 

    use lib '.';

or 

    use FindBin '$Bin';
    use lib "$Bin";

**Comment by syohex** [comment]
Thanks for reporting issue. I have merged #3.

**Comment by benkasminbullock** [comment]
Thanks, although I can't get this module to even print an error message when I try to install it. I run "perl Build.PL" and it just exits without any error message. When I use cpanm it just fails with a message "N/A".

**Comment by syohex** [comment]
I suppose you need to install [aalib](http://aa-project.sourceforge.net/aalib/) before installing this module. However error message should be improved on such platform.

**Comment by benkasminbullock** [comment]
Thank you for a very prompt response.

The reason I am querying about this module is that I have written a module [Image::PNG::Libpng](https://metacpan.org/release/Image-PNG-Libpng) which includes a script `pnginspect` to inspect PNG files at a terminal. I wanted to add some kind of ASCII output of the PNG file if it's available. I was looking on CPAN for something and found your module. I'm not sure if it's possible for me to add that to my script pnginspect, I am just exploring options at the moment.

I downloaded aalib but I did not install it, I was hoping that I could put it into user space rather than into a system directory, but this module did not say what to do.
